src/blog/2023/06/dashboard-announcement.md

```diff
@@ -0,0 +1,42 @@
+<!--more-->
+We at FlowForge have identified a significant need for a modern, interactive data visualization and dashboard solution. In response, we've decided to embark on an exciting journey: the creation of the official successor to the Node-RED Dashboard.
+
+## The Problem at Hand
+
+The original Node-RED Dashboard is based on Angular v1, which is no longer maintained. Although small patches have been and will continue to be applied on a "best can do" basis, there will be no major feature upgrades. The lack of ongoing maintenance and updates has the potential to lead to underlying security breakages, a risk we are not comfortable taking. We have recognized the need to innovate and adapt, which is why we are creating a completely new project to replace the existing Node-RED Dashboard.
+
+## The Solution
+
+The successor to the Node-RED Dashboard will be a completely new project, published as a separate package. This new project will take the reins from the old dashboard and guide us into the future. However, the Node-RED community can rest assured that the project will stay under the Apache 2.0 licence. This ensures that the core principles of open-source and community-driven development will remain intact.
+
+In line with our open core approach, we will provide vendors the opportunity to build premium features around the core. This allows businesses to customize and enhance their experience with the new dashboard, while still maintaining the integrity of the open-source project.
+
+At FlowForge, we are not just creating a new dashboard, but also adding an Enterprise Extension with additional functionalities. This extension will provide features and capabilities geared towards the needs of larger organizations and enterprises.
+
+## Community Involvement and Open Source Contribution
+
+We believe in the power of the community, and we want your feedback. If there are features you'd like to see or improvements you think can be made, we invite you to open a [Github issue](https://github.com/flowforge/flowforge-nr-dashboard/issues/new/choose). Your insights and suggestions will be invaluable in shaping the future of this project. Contributions from the community are not just welcome, but highly encouraged. We believe that the strength of a project is proportional to the strength of its community.
+
+In our commitment to transparency and collaboration, we will be documenting all major decision-making processes and sharing the details here in our blog as they become available. By working closely with additional Node-RED key figures like Dave Conway Jones, and by making our development process as open and collaborative as possible, we aim to ensure that the successor to the Node-RED Dashboard lives up to the high standards set by the original, while also introducing innovative features and enhancements.
```

Comment:
It feels like the fact of working with Dave CJ is an important fact since he was the original developer. I almost feel like this should be moved up to one of the first two paragraphs.

Reply:
> since he was the original developer.

He wasn't the original developer, but DCJ, Nick and I took it off the original developer, and DCJ has been primary owner for past 6 years or so.

Reply:
I don't think that changes the point @iskerrett is making here; in the eyes of the community, DCJ is the primary maintainer of dashboard today. We should highlight this is being done in consultation with him early on.